internals/scripts/setup.js

```diff
@@ -29,10 +28,47 @@ cleanRepo(function () {
 });
 
 /**
- * Deletes the .git folder in dir
+ * Deletes the .git folder in dir only if cloned from our repo
  */
 function cleanRepo(callback) {
-  shell.rm('-rf', '.git/');
+  fs.stat('.git', function(err, stats) {
```

Comment:
* Initially I had checking for `.git` folder and `config` file because I was logging onto the console why I'm doing blind `git init`, I can remove that.
* About removing `|| []` is a foolproof way for me to ensure that it won't fail when there are no urls to be found in your config. It is a possible edge case. I don't think it should be removed